src/Parameters.jl

```diff
@@ -139,5 +139,11 @@ cv_v(ps::CMPS) = TD.Parameters.cv_v(thermodynamics_params(ps))
 cv_l(ps::CMPS) = TD.Parameters.cv_l(thermodynamics_params(ps))
 cv_i(ps::CMPS) = TD.Parameters.cv_i(thermodynamics_params(ps))
 
+# Temporary SB2001 2M parameters (should move to CLIMAParameters when all is ready)
+kc_SB2001(ps::CMPS) = 9.44e9 # m^3 kg^-2 s^-1 
+kr_SB2001(ps::CMPS) = 5.78 # m^3 kg^-2 s^-1
+xstar_SB2001(ps::CMPS) = 2.6e-10 # kg
+ν_SB2001(ps::CMPS) = 2
```

Comment:
So the plan would be to first merge it like this and then move the parameters into CLIMAParameters?

Reply:
@trontrytel 
Thanks for the review
No, I wanted to wait until the first round of review is done, because I wasn't sure if there should be other parameters (for example the ones for the universal function) that need to be added to CliMAParameters too. Now, I'll remove these functions and create a pull request to CliMAParameters.